CastReporting.Domain/DataObject/RuleDetails.cs

```diff
@@ -10,7 +10,7 @@ public class RuleDetails
         public string Href { get; set; }
 
         [DataMember(Name = "key")]
-        public string Key { get; set; }
+        public Int32 Key { get; set; }
```

Comment:
You should set Int32? instead of Int32 to avoid exception if there is no value. We had this issue several times in the past